Assets/MRTK/Services/InputSystem/FocusProvider.cs

```diff
@@ -43,6 +43,8 @@ public class FocusProvider : BaseCoreSystem,
         {
             maxQuerySceneResults = profile.FocusQueryBufferSize;
             focusIndividualCompoundCollider = profile.FocusIndividualCompoundCollider;
+            inputSystemProfile = profile;
+            shouldUseGraphicsRaycast = inputSystemProfile == null || inputSystemProfile.ShouldUseGraphicsRaycast;
```

Comment:
if we do not have an input system profile, we cannot run, correct?

Reply:
True 
![image](https://user-images.githubusercontent.com/3580640/117067959-842a9600-acdf-11eb-99c4-557e140a804c.png)

Reply:
True 
![image](https://user-images.githubusercontent.com/3580640/117067997-90aeee80-acdf-11eb-9cf9-6124eae56a4e.png)

Reply:
so, do we need the inputSystemProfile == null part in the shouldUseGraphicsRaycast setter?

Reply:
Because we try to read from the profile in this same line, I'd say yes. We should account for it potentially not existing here and let the existing start-up code catch if we're in an unsupported state.